edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/IModuleIdentity.cs

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Microsoft. All rights reserved.
 namespace Microsoft.Azure.Devices.Edge.Agent.Core
 {
+    using Microsoft.Azure.Devices.Edge.Util;
+
     public interface IModuleIdentity
     {
         string IotHubHostname { get; }
 
-        string GatewayHostname { get; }
+        Option<string> ParentEdgeHostname { get; }
```

Comment:
I think this property name is self-explained.  If I put environment variable name in property description, it will be missed to update when environment variable name is changed. Therefore it is good for now.